UnitTestFramework.brs

```diff
@@ -358,6 +358,33 @@ Function BTS__AssertAAHasKeys(array as dynamic, keys as object, msg = "" as stri
     return ""
 End Function
 
+'----------------------------------------------------------------
+' Fail if the AA doesn't contain equal values
+'
+' @param expected (object) AA with expected key:value's.
+' @param actual (object) AA to test against.
+' @param msg (string) An error message.
+' Default value: ""
+'
+' @return An error message.
+'----------------------------------------------------------------
+Function BTS__AssertAAContains(expected as object, actual as object, msg = "" as string) as string
+    if TF_Utils__IsAssociativeArray(expected) and TF_Utils__IsAssociativeArray(actual)
+        for each key in expected
+            if not actual[key] = expected[key]
```

Comment:
IMO it could be better if you use `m.eqValues(actual[key], expected[key])` (see `BTS__AssertEqual` as reference) to compare 2 dynamic values.